projects/packages/my-jetpack/_inc/components/product-cards-section/backup-card/index.jsx

```diff
@@ -119,20 +112,128 @@ const NoBackupsValueSection = ( { siteData } ) => {
 	);
 };
 
-// eslint-disable-next-line no-unused-vars
+const WithBackupsValueSection = ( { lastUndoableEvent } ) => {
+	if ( ! lastUndoableEvent || ! lastUndoableEvent.data ) {
+		return null;
+	}
+	const { last_rewindable_event: lastRewindableEvent = {} } = lastUndoableEvent.data;
+
+	return (
+		<div className={ styles.activity }>
+			<Gridicon icon={ lastRewindableEvent.gridicon } size={ 24 } />
+			<p className={ styles.summary }>{ lastRewindableEvent.summary }</p>
+		</div>
+	);
+};
+
+const getTimeSinceLastRenewableEvent = lastRewindableEventTime => {
+	if ( ! lastRewindableEventTime ) {
+		return '';
+	}
+
+	const now = new Date();
+	const lastRewindableEventDate = new Date( lastRewindableEventTime );
+	const timeSinceLastRenewableEvent = now - lastRewindableEventDate;
+
+	if ( timeSinceLastRenewableEvent > 0 ) {
+		const days = Math.floor( timeSinceLastRenewableEvent / ( 1000 * 60 * 60 * 24 ) );
+		const hours = Math.floor(
+			( timeSinceLastRenewableEvent % ( 1000 * 60 * 60 * 24 ) ) / ( 1000 * 60 * 60 )
+		);
+		const minutes = Math.floor(
+			( timeSinceLastRenewableEvent % ( 1000 * 60 * 60 ) ) / ( 1000 * 60 )
+		);
+		const seconds = Math.floor( ( timeSinceLastRenewableEvent % ( 1000 * 60 ) ) / 1000 );
+
+		if ( days > 0 ) {
+			return sprintf(
+				// translators: %s is the number of days since the last backup
+				_n( '%s day ago', '%s days ago', days, 'jetpack-my-jetpack' ),
+				days
+			);
+		}
+
+		if ( hours > 0 ) {
+			return sprintf(
+				// translators: %s is the number of hours since the last backup
+				_n( '%s hour ago', '%s hours ago', hours, 'jetpack-my-jetpack' ),
+				hours
+			);
+		}
+
+		if ( minutes > 0 ) {
+			return sprintf(
+				// translators: %s is the number of minutes since the last backup
+				_n( '%s minute ago', '%s minutes ago', minutes, 'jetpack-my-jetpack' ),
+				minutes
+			);
+		}
+
+		return sprintf(
+			// translators: %s is the number of seconds since the last backup
+			_n( '%s second ago', '%s seconds ago', seconds, 'jetpack-my-jetpack' ),
+			seconds
+		);
+	}
+};
+
 const BackupCard = ( { admin, productData, fetchingProductData } ) => {
 	const slug = 'backup';
 
+	const { recordEvent } = useAnalytics();
 	const { detail } = useProduct( slug );
 	const { status } = detail;
 	const hasBackups = status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
 
-	const { site_data: siteData = {} } = productData || {};
+	const { site_data: siteData = {}, last_undoable_event: lastUndoableEvent = {} } =
+		productData || {};
+
+	const lastRewindableEventTime = lastUndoableEvent?.data?.last_rewindable_event?.published;
+	const hasRewindableEvent = hasBackups && lastUndoableEvent?.data;
+	const undoBackupId = lastUndoableEvent?.data?.undo_backup_id;
+
+	const handleUndoClick = () => {
+		recordEvent( 'jetpack_myjetpack_backup_card_undo_click', {
+			product: slug,
+			undoBackupId,
```

Comment:
I believe Tracks property names need to be snake case.  Calypso enforces it, shows a console warning and refuses to send the event. I'm not sure about Jetpack, I'm not seeing a console error, but looking through the codebase all the ones I see are snake_case.

```suggestion
			undo_backup_id: undoBackupId,
```